### PR TITLE
Allow to set default app

### PR DESCRIPTION
**Pull request by stsewd**
Hi, I always find myself doing `:Pickachu date`, so I think this needs to be configurable.

### CODE REVIEW

File: plugin/pickachu/main.py [hunk 1]
```diff
@@ -8,22 +8,23 @@
...
 	arglength = int(vim.eval('a:0'))
-	CHOOSEN_APP = 'color'
+	choosen_app = DEFAULT_APP
 	CHOOSEN_FORMAT = None
 	if arglength > 0:
-		CHOOSEN_APP = vim.eval('a:1')
+		choosen_app = vim.eval('a:1')
 	if arglength > 1:
 		CHOOSEN_FORMAT = vim.eval('a:2')
-	if not CHOOSEN_APP:
```
Comment (stsewd) [inline plugin/pickachu/main.py]: I'm not sure about this, since I'm not a vim-plugin expert, but at first glance it seems redundant

**Comment by DougBeney** [comment]
Greatly appreciate the contribution!
Two revisions and I think we'll be good to go:
- Use tabs, not spaces
- Change instances of `choosen_app` to `CHOOSEN_APP`

And you're right about the redundant if-statement. I had to do some testing to make sure I didn't have it there for a reason.

**Comment by stsewd** [comment]
@DougBeney can we follow PEP8 style guide? 

- http://pep8.org/#tabs-or-spaces
- http://pep8.org/#constants

I will change the PR if you don't want that, though.

**Comment by DougBeney** [comment]
I'm aware of pep8. I just prefer tabs ;)

**Comment by stsewd** [comment]
@DougBeney hope now it's ok

**Comment by DougBeney** [comment]
Accepted. Thanks for contributing!

**Pull request merged**